### ISSUE TITLE
Intro to the project + rename helper.js to utils.js

### ISSUE THREAD

**Issue by atherdon** [GroceriStar/groceristar-fetch#336]
You need to read readme file: https://github.com/GroceriStar/groceristar-fetch/blob/master/README.md
it also contains links to articles, where I explaining some details. If something will be not clear - don't hesitate and ask questions.

Tasks, that you can take(feel free to ask clarification if something is not clear)
1. rename helper.js to utils.js
**note** it's not that easy as it can look like. take time and pay attention. 

@Beni03

**Comment by Beni03** [comment]
I changed the file name. Is it what you expected?

**Comment by atherdon** [comment]
nope, i send you my comments

**Comment by Beni03** [comment]
updated all the modules( under src/project) which uses helper.js file

**Comment by atherdon** [comment]
did you push that changes via pull request?

**Comment by atherdon** [comment]
i don't see your changes here: https://github.com/GroceriStar/groceristar-fetch/pull/339

**Comment by Beni03** [comment]
Sorry! Yeah now I pushed all the changes via pull request

**Comment by atherdon** [comment]
no problems. awesome progress, btw
Ok, so looks like this task is done, can you confirm that it works well and we can move forward?
Btw, i saw that you saw few of my @todos - you update them. Maybe you can deal with that simple tasks too?

**Comment by Beni03** [comment]
Yes, we can move forward.

Thanks,
Benitha

**Comment by atherdon** [comment]
cool. then please deal with todos that you've changed

**Comment by atherdon** [comment]
here when i saw it: https://github.com/GroceriStar/groceristar-fetch/pull/345/commits/44d1f30059465ecbc061f304a42deb1e7c64502b

**Comment by Beni03** [comment]
Sure. Will work on it.

**Comment by Beni03** [comment]
Can I use crypto module to generate random IDs?

**Comment by Beni03** [comment]
Please ignore the above question.

**Comment by Beni03** [comment]
const utils = require('./utils');

const getFiveRandomId = function () {
return [
utils. __generateId(),
utils.__generateId(),
utils.__generateId(),
utils.__generateId(),
utils.__generateId()
]
}

var result = getFiveRandomId();
console.log(result);

is this what you  expected? looks like it is done already.

**Comment by atherdon** [comment]
maybe it works, but i still think that this current approach is not ideal. i think by combining some lodash method with `generateId` we can make it better

**Comment by Beni03** [comment]
I have added the lodash method to generate unique ID

**Comment by atherdon** [comment]
yeah, take alook at this comment: https://github.com/GroceriStar/groceristar-fetch/pull/354#issuecomment-485362634

**Comment by Beni03** [comment]
Yes, it’s possible to generate Id using lodash.  But, I think the generated Id’s will look like auto increment key values. Is that what we want?

**Comment by atherdon** [comment]
@vadim9999 please jump here too.

**Comment by vadim9999** [comment]
I think generate Id using lodash is for simple task and easy to check what is relationship beetwen  objects. uuidv is for more complex task. I think it more secure. Hacker should know this randomly generated id with characters and numbers. Also max value of number is limited but what he will do when max limit will be reach? In simple DB like Hello Wrld I used just increment numbers but it should be unique. Key is like a cipher to identify user. Here we are using objects
